TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/approve.py

```diff
@@ -0,0 +1,205 @@
+                    argo_server=argo_server,
+                    token=token,
+                    insecure=insecure
+                )
+
+                if all_workflows_result['success'] and all_workflows_result['count'] > 0:
+                    # Workflows exist but none need approval
+                    click.echo(
+                        f"No workflows require approval in namespace {namespace}.\n"
+                        f"Use 'workflow status' to see workflow details.",
+                        err=True
+                    )
+                else:
+                    # No workflows exist at all
+                    click.echo(f"Error: No workflows found in namespace {namespace}", err=True)
+                ctx.exit(ExitCode.FAILURE.value)
+            elif list_result['count'] > 1:
+                workflows_list = ', '.join(list_result['workflows'])
+                click.echo(
+                    f"Error: Multiple workflows found. Please specify which one to approve.\n"
+                    f"Found workflows: {workflows_list}",
+                    err=True
+                )
+                ctx.exit(ExitCode.FAILURE.value)
+
+            workflow_name = list_result['workflows'][0]
+            click.echo(f"Auto-detected workflow: {workflow_name}")
+
+        # Get workflow status to display details
+        status_result = service.get_workflow_status(
+            workflow_name=workflow_name,
+            namespace=namespace,
+            argo_server=argo_server,
+            token=token,
+            insecure=insecure
+        )
+
+        if not status_result['success']:
+            click.echo(f"Error getting workflow status: {status_result['error']}", err=True)
+            ctx.exit(ExitCode.FAILURE.value)
+
+        # Display workflow information
+        click.echo("\n" + "=" * 60)
+        click.echo(f"Workflow: {workflow_name}")
+        click.echo(f"Namespace: {namespace}")
+        click.echo(f"Phase: {status_result['phase']}")
+        if status_result['started_at']:
+            click.echo(f"Started: {status_result['started_at']}")
+        click.echo("=" * 60)
+
+        # Display steps and collect suspended steps
+        suspended_steps = []
+        if status_result['steps']:
+            click.echo("\nWorkflow Steps:")
+            click.echo("-" * 60)
+            for step in status_result['steps']:
+                step_type = step['type']
+                step_name = step['name']
+                step_phase = step['phase']
+
+                # Collect suspended steps for approval selection
+                if step_type == 'Suspend' and step_phase == 'Running':
+                    suspended_steps.append(step_name)
+                    click.echo(f"  > {step_name} [{step_type}] - {step_phase} (WAITING FOR APPROVAL)")
+                else:
+                    status_icon = "+" if step_phase == "Succeeded" else "*"
+                    click.echo(f"  {status_icon} {step_name} [{step_type}] - {step_phase}")
```

Comment:
why is approve showing a bunch of other results - this will include items that aren't suspends too - right?  This might make it harder to convert the overall output into a machine readable format that's easily consumable downstream.